### ISSUE TITLE
TypeError: hammerTime is undefined 	 with multiple modals

### ISSUE THREAD

**Issue by ibrainventures** [likeastore/ngDialog#209]
Hi,

with multiple modals i got on the close for the last (btw. first opend modal) : the console error:

TypeError: hammerTime is undefined
    hammerTime.off('tap', closeByDocumentHandler);

and there is no possibility to close the modal ..

if i deactivate hammer.js it is working ..

any idea ?

**Comment by csmithmaui** [comment]
I have this same problem and created a [plnkr](http://embed.plnkr.co/ECUlb3/preview) to illustrate the behavior.  Closing the first level dialog works without hammer.js included in a script tag.  In the plnkr, uncomment the hammer.js script tag and then refresh, you will see that the original dialog will not close after opening the inner(nested) one.

It seems like it has to do with the inner dialog overriding the performCloseDialog function with it's scope.  The scope hash for the first dialog is the same as the scope hash for the second dialog after closing the second, When trying to access scope.hammerTime of the first dialog after closing the second dialog, it's already been wiped out.

Anyway, I'm not very familiar with ngDialog, because I just started using it, so I don't really have an idea of the best way to fix it.  It would be nice if it was working though because I love everything else about it.
